templates/prioritization_agenda.tt

```diff
@@ -15,19 +15,13 @@ tags: weekly, rustc
 - Old MCPs (not seconded, take a look)
 {{-issues::render(issues=mcp_old_not_seconded, indent="  ", empty="No old proposals this time.")}}
 - Pending FCP requests (check your boxes!)
-{{-issues::render(issues=in_pre_fcp_compiler_team, indent="", newline_and_indent=false, empty="")}}
-{{-issues::render(issues=in_pre_fcp_rust, indent="", newline_and_indent=false, empty="")}}
-{{-issues::render(issues=in_pre_fcp_forge, indent="  ", empty="No pending FCP requests this time.")}}
+{{-issues::render(issues=in_pre_fcp, indent="  ", empty="No pending FCP requests this time.")}}
```

Comment:
The FCP template now is one element (each FCP status) with all the issues together, so I can effortlessy remove empty lines. Only issue here, we might lose the sorting (compiler_team, rust, forge). But I think we can live without it.

Reply:
I don't think we will lose the sorting.